src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -59,7 +58,7 @@ import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
 import { ConfirmDialog } from '@/components/dialogs'
 
 // Interfaces
-import { ConfirmDialogType } from '@/interfaces'
+import { ConfirmDialogType, FormType } from '@/interfaces'
```

Comment:
You don't use FormType here, but you could instead of HTMLFormElement on line 72.

Reply:
Removed